src/public/services/AdminFormService.ts

```diff
@@ -70,6 +70,17 @@ export const updateCollaborators = async (
     .then(({ data }) => data)
 }
 
+export const duplicateSingleFormField = async (
+  formId: string,
+  fieldId: string,
+): Promise<FormFieldDto[]> => {
+  return axios
+    .post<FormFieldDto[]>(
```

Comment:
I'm not sure if this typing is correct. The API returns a single form field, not an array, right?